CONTRIBUTING.md

```diff
@@ -0,0 +1,50 @@
+How to Contribute
+=================
+
+Getting help
+------------
+To ask questions about the XBee Library for Android go to the
+[Digi Forum](http://www.digi.com/support/forum).
+
+
+Reporting a bug
+---------------
+The way to report bugs is to use the
+[GitHub issue tracker](http://github.com/digidotcom/XBeeAndroidLibrary/issues).
+Before reporting a bug, please read the following points:
+
+1. Make sure that is really a bug by checking the
+[documentation](http://docs.digi.com/display/XBJLIB) and consulting the
+[FAQs](https://docs.digi.com/display/XBJLIB/Frequently+Asked+Questions+%28FAQs%29)
+section.
+2. If you still think you have found a bug, make sure someone has not already
+reported it. See the list of
+[known issues](http://github.com/digidotcom/XBeeJavaLibrary/issues).
+3. If it has not been reported yet, create a new issue. Make sure to add enough
+detail so that the bug can be reproduced.
+
+**Note**: The issue tracker is for bugs, not requests for help. Questions
+should be asked on the [Digi Forum](http://www.digi.com/support/forum) instead.
+
+
+Suggesting a new feature
+------------------------
+1. Consult the [FAQs](https://docs.digi.com/display/XBJLIB/Frequently+Asked+Questions+%28FAQs%29)
```

Comment:
Replace this link with this one: https://www.digi.com/resources/documentation/digidocs/90001438/#reference/r_xb_java_lib_faq.htm